extension/EdFi.Ods.Extensions.LMSX/Artifacts/Schemas/EXTENSION-Interchange-Descriptors-Extension.xsd

```diff
@@ -58,7 +59,6 @@ The Descriptor interchange is used to describe metadata about the descriptors an
         <xs:element name="CredentialTypeDescriptor" type="CredentialTypeDescriptor" />
         <xs:element name="CreditCategoryDescriptor" type="CreditCategoryDescriptor" />
         <xs:element name="CreditTypeDescriptor" type="CreditTypeDescriptor" />
-        <xs:element name="CTEProgramServiceDescriptor" type="CTEProgramServiceDescriptor" />
```

Comment:
I built this in Linux instead of Windows, because NodeJs is broken in Windows on my computer. The two treat sorting differently; Windows puts "CT" after "Cr", but Linux treats the upper case letters differently, so "CT" is before "Ca".